contrib/epee/include/math_helper.h

```diff
@@ -251,12 +251,23 @@ namespace math_helper
 #endif
     }
 
+    void set_next_interval()
+    {
+      m_interval = m_interval_min;
+      if (m_interval_max > m_interval_min)
+      {
+        const float r = (rand() % 65536) / 65536.f /* never reach 1 */; // good enough ?
```

Comment:
rand return values between 0 and RAND_MAX, and RAND_MAX is not guaranteed to be more than 32767